SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using SeleniumPracticalExercise.PageObjects.Common;
+
+namespace SeleniumPracticalExercise.PageObjects
+{
+    public class EmployeeInformationPage : BasePageLocal
+    {
+        private readonly By _addButtonLocator = By.CssSelector("div.orangehrm-header-container > button");
+        private readonly By _employeeIdFieldLocator = By.CssSelector("div.oxd-input-group input.oxd-input");
+        private readonly By _firstNameResultsLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][3]/div");
+        private readonly By _idResultsLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][2]/div");
+        private readonly By _lastNameResultsLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][4]/div");
+        private readonly By _searchButtonLocator = By.CssSelector("button[type='submit']");
+
+        public EmployeeInformationPage(IWebDriver driver) : base(driver)
+        {
+        }
+
+        /// <summary>
+        /// Clicks the Add Button
+        /// </summary>
+        public void ClickAdd()
+        {
+           Click(_addButtonLocator);
+        }
+
+        /// <summary>
+        /// search for employee ID field and insert stored ID
+        /// then we click search button
+        /// </summary>
+        /// <param name="storeEmployeeID"></param>
+        public void SetEmployeeIdField(string storeEmployeeID)
```

Comment:
This method name no longer matches what it's doing.

Reply:
I renamed it to SetEmployeeId() 
but I am a little confused why you said the method name does not match what its doing 
since it is dealing with the EmployeeId
Was it that word Field I added, since we are grabbing the id and setting it to storeEmployeeId, we don't need the word Field?

Reply:
This method is searching for an employee. Don't look at the individual steps, look at what it's doing holistically.

The method name could be 'EmployeeSearch' or 'EmployeeSearchById` or if employee search is the only search type possible it could be shortened to 'Search' or 'SearchById'. I would go with 'SearchById' because employee search is the only search possible. It's also on the employee information page so employee search is at least implied. Also, I can imagine that there will likely be other types of employee searches using other fields so 'SearchById' already makes room for 'SearchByName' or 'SearchByTeam' or whatever other methods there might be as well as the generic 'Search' which would take all fields as optional parameters, letting you use any combination of search parameters.